core/src/main/java/com/sequenceiq/cloudbreak/service/rdsconfig/RdsConfigService.java

```diff
@@ -146,15 +146,20 @@ private void delete(RDSConfig rdsConfig) {
     private void checkRdsConfigNotAssociated(RDSConfig rdsConfig) {
         List<Cluster> clustersWithProvidedRds = new ArrayList<>(clusterRepository.findAllClustersByRDSConfig(rdsConfig.getId()));
         if (!clustersWithProvidedRds.isEmpty()) {
-            StringBuilder clusters = new StringBuilder();
-            clusters.append('[');
-            for (int i = 0; i < clustersWithProvidedRds.size() - 1; i++) {
-                clusters.append(clustersWithProvidedRds.get(i).getName()).append(", ");
+            if (clustersWithProvidedRds.size() > 1) {
```

Comment:
!isempty()?

Reply:
like in other cases, different message come back if the related clusters size is *more* than one